doc/user/layouts/shortcodes/session-variables.html

```diff
@@ -20,6 +20,7 @@
 emit_timestamp_notice                       | `false`                   | Boolean flag indicating whether to send a `notice` specifying query timestamps.
 emit_trace_id_notice                        | `false`                   | Boolean flag indicating whether to send a `notice` specifying the trace ID, when available.
 enable_session_rbac_checks                  | `false`                   | **Read-only.** Boolean flag indicating whether RBAC is enabled for the current session.
+enable_per_replica_notice                   | `true`                    | Whether to print a notice when querying replica introspection relations.
```

Comment:
Looking above, it seems other notice variables follow the `emit_*_notice` pattern, so I think this one should too? Also we should make the description text consistent.

Seeing this name I assumed this alerts on replica-targeted queries. But actually it alerts on introspection queries. I think the name should reflect that. How about `emit_introspection_query_notice`?